scripts/GraviSuite.zs

```diff
@@ -113,9 +111,13 @@ recipes.addShaped(<GraviSuite:epicLappack:27>, [
 
 // --- Relocator
 recipes.addShaped(<GraviSuite:relocator:27>, [
-[<gregtech:gt.metaitem.01:32676>, <dreamcraft:item.NeutroniumItemCasing>, <gregtech:gt.metaitem.01:32676>],
-[<ore:circuitUltimate>, <IC2:blockMachine2>, <ore:circuitUltimate>],
-[<dreamcraft:item.NeutroniumItemCasing>, <IC2:itemBatLamaCrystal:*>, <dreamcraft:item.NeutroniumItemCasing>]]);
+[<ore:screwOsmium>,<ore:craftingToolScrewdriver>,<ore:screwOsmium>],
+[<ore:plateOsmium>,<DraconicEvolution:teleporterMKII>,<ore:plateOsmium>],
```

Comment:
That makes no sense, DE teleporter is in all respects better than personal relocator. It requires no energy, and stores unlimited amount of coordinates.

Reply:
point for you. Maybe we need to re-nerf it again? @bartimaeusnek

Reply:
@repo-alt  "However, it has one downside of requiring Ender Pearls for teleportation in the ratio of 1 Pearl per teleportation." So i would say using energy instead of Ender Pearls is an upgrade.